src/Routers/FilesRouter.js

```diff
@@ -244,9 +244,8 @@ export class FilesRouter {
       }
       // run afterSaveFile trigger
       await triggers.maybeRunFileTrigger(triggers.Types.afterSave, fileObject, config, req.auth);
-      res.status(201);
+      res.status(201).json(saveResult);
```

Comment:
Why are you retuning `saveResult` directly?  The code before your change has only been returning a `Location` key?